src/Tools.pm

```diff
@@ -694,7 +694,7 @@ sub GetSysconfigValue
 
   if(open my $fh, Bootloader::Path::Sysconfig()) {
     while(<$fh>) {
-      $val = $1, last if /^\s*$key\s*=\s*"(.*)"\s*$/;
+      $val = $1, last if /^\s*$key\s*=\s*"?(.*?)"?\s*$/;
```

Comment:
How about `/^\s*$key\s*=\s*+(["']?+)(.*)\1\s*$/` excluding at least some lines with mismatched quotes?

Reply:
updated it with one less `+` that should not matter as it is not captured.